### PR TITLE
Feat/st 1106 2: Updated few roles to Case events

### PR DESCRIPTION
**Pull request by sjamihmcts**
### Change description ###

Updated few roles to Case events

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-1106

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java [hunk 1]
```diff
@@ -94,7 +95,7 @@ private void doConfigure(final ConfigBuilder<CaseData, State, UserRole> configBu
                 .submittedCallback(this::partiesContacted)
                 .grant(CREATE_READ_UPDATE, SUPER_USER,
                     ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER, ST_CIC_HEARING_CENTRE_ADMIN,
-                    ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE)
+                    ST_CIC_HEARING_CENTRE_TEAM_LEADER, ST_CIC_SENIOR_JUDGE, ST_CIC_RESPONDENT)
                 .grantHistoryOnly(
```
Comment (ElifDenizEsea) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java:99]: There is another event Called RespondentContactParties. Would not be an issue?

**Review by osheaf01 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java (hunk 1 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerContactParties.java:99]: ok Elif, haven't realized that there is a separate class for Respondent. So this is change is not needed anymore.

**Review by osheaf01 — DISMISSED** [review]
LGTM

**Review by Shailesh-walke — APPROVED** [review]

**Pull request merged**